Messaging/Forward Messages - NodeJS/.env.sample

```diff
@@ -0,0 +1,4 @@
+PROJECT_ID=<YOUR PROJECT ID>
+API_TOKEN=<YOUR API TOKEN>
+ORIGINAL_NUMBER=<YOUR SIGNALWIRE NUMBER RECEIVING MESSAGES IN E.164 FORMAT (+1XXXYYYZZZZ)>
+TO_NUMBER=<NUMBER YOU WOULD LIKE TO FORWARD TO IN E.164 FORMAT(+1XXXYYYZZZZ)>
```

Comment:
sounds good. I still think it would be a good idea to create a style guide wiki for this guides repo similar to the one you created for the signalwire-docs repo

Reply:
We definitely need that!